src/Features/CSharp/Portable/CodeFixes/GenerateMethod/GenerateMethodCodeFixProvider.cs

```diff
@@ -80,6 +83,12 @@ protected override SyntaxNode GetTargetNode(SyntaxNode node)
                 return memberBindingExpression.Name;
             }
 
+            var assignmentExpression = node as AssignmentExpressionSyntax;
```

Comment:
Are you sure? It could be that rather than choosing a method with the wrong
signature to add to the event that the wrong event had been chosen.

On Mon, 8 Aug 2016 17:08 Kevin Pilch-Bisson, <notifications@github.com>
wrote:

> In
> src/Features/CSharp/Portable/CodeFixes/GenerateMethod/GenerateMethodCodeFixProvider.cs
> <https://github.com/dotnet/roslyn/pull/12977#discussion_r73903609>:
>
> > @@ -80,6 +83,12 @@ protected override SyntaxNode GetTargetNode(SyntaxNode node)
> >                  return memberBindingExpression.Name;
> >              }
> >
> > +            var assignmentExpression = node as AssignmentExpressionSyntax;
>
> That sounds like a bug in the span being reported by the compiler -
> @jaredpar <https://github.com/jaredpar>?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dotnet/roslyn/pull/12977/files/5f4832d3d3046ca8fbc428d32fed2980e1938493#r73903609>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA822tneAUnNK522AGVuHtb1sq1vl51Dks5qd1RugaJpZM4JeRUh>
> .
>